.github/workflows/publish_helm_chart.yaml

```diff
@@ -0,0 +1,44 @@
+# package and push the zenml helm chart to the ECR repository
+name: Publish Helm Chart
+
+on: workflow_call
+
+jobs:
+  publish_to_ecr:
+    name: Publish Helm Chart 🛞 to ECR ☁️
+    runs-on: ubuntu-latest
+    env:
+      ZENML_DEBUG: 1
+      ZENML_ANALYTICS_OPT_IN: false
+      PYTHONIOENCODING: 'utf-8'
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      # add tag version to Chart.yaml file for variable version
+      - name: Add tag version to Chart.yaml
+        run: |
+          sed -i "s/version: .*/version: \"${{ github.ref_name }}\"/" src/zenml/zen_server/deploy/helm/Chart.yaml
```

Comment:
When it comes to commands like this, I like it when they are commented a bit verbosly:

# Finds like that contains "version: " and replaces the complete line with "version: ___"

Reply:
Didin't mean to scream it :D

Reply:
ahaha, point taken